new-pipeline/crash-summary.html

```diff
@@ -0,0 +1,200 @@
+                <li>
+                    Total time is inconsistent across OSes. On Windows,
+                    total time includes sleep time. All other OSes
+                    total time does not include sleep time.
+                </li>
+                <li>Dashboard status: BETA. Contact bsmedberg with questions.</li>
+            </ul>
+        </p>
+
+        <h2 id="view">
+            <span class="dropdown">
+                <a id="channel-selector" class="dropdown-toggle"
+                   data-toggle="dropdown" data-target="#"
+                   aria-haspopup="true" aria-expanded="false">
+                    Release Channel: <span id="channel-value">nightly</span>
+                    <span class="caret"></span>
+                </a>
+                <ul id="channel-dropdown" class="dropdown-menu" aria-labelledby="channel-selector">
+                    <li><a href="#" data-channel="nightly">nightly</a></li>
+                    <li><a href="#" data-channel="aurora">developer edition (aurora)</a></li>
+                    <li><a href="#" data-channel="beta">beta</a></li>
+                    <li><a href="#" data-channel="release">release</a></li>
+                </ul>
+            </span>
+            <span class="dropdown">
+                <a id="sort-selector" class="dropdown-toggle"
+                   data-toggle="dropdown" data-target="#"
+                   aria-haspopup="true" aria-expanded="false">
+                    Sort by: <span id="sort-value" data-sort="buildid">Build ID</span>
+                    <span class="caret"></span>
+                </a>
+                <ul id="sort-dropdown" class="dropdown-menu" aria-labelledby="sort-selector">
+                    <li><a href="#" data-sort="buildid">Build ID</a></li>
+                    <li><a href="#" data-sort="date">Calendar Date</a></li>
+                </ul>
+            </span>
+        </h2>
+
+        <div id="hover-details" class="invisible">
+            <dl>
+                <dt id="detail-sort-label">_</dt>
+                <dd id="detail-sort-value"></dd>
+                <dt>Hours</dt>
+                <dd id="detail-hours"></dd>
+            </dl>
+            <h3>Main Process</h3>
+            <dl>
+                <dt>Aborts</dt>
+                <dd><div class="ib">
+                    <span id="detail-mainaborts"></span><br>
+                    <span id="detail-mainaborts-rate"></span>/khour
+                </div></dd>
+                <dt>Crashes</dt>
+                <dd><div class="ib">
+                    <span id="detail-maincrashes"></span><br>
+                    <span id="detail-maincrashes-rate"></span>/khour<br>
+                    <span id="detail-main-detection-rate"> detection rate
+                </div></dd>
+                <dt>Submission Rate</dt>
+                <dd id="detail-main-submission-rate"></dd>
+            </dl>
+
+            <h3>Plugins</h3>
+            <dl>
+                <dt>NPAPI Aborts</dt>
```

Comment:
Unfortunately in this pass I can't add links because you can't get to them with the rollover. Still working with Ali/Hamilton on making that better.